nms/app/packages/magmalte/app/components/context/SubscriberContext.js

```diff
@@ -42,7 +43,7 @@ export type SubscriberContextType = {
   gwSubscriberMap: {[gateway_id]: Array<subscriber_id>},
   setState?: (
     key: string,
-    val?: mutable_subscriber,
+    val?: mutable_subscriber | mutable_subscribers,
```

Comment:
Lets just use mutable_subscribers and pass in array of mutable subscriber in other locations to keep this consistent.

Reply:
I think it would not work because we only have bulk add api on the swagger api, apart from that the other ones like edit work on one subscriber at a time. Also, doing this change would lead to complications like if a user passes an array of some edited and some new added because the bulk add endpoint returns error for those cases. So, I think it is better to only allow an array to be passed when we are actually using it to call the bulk add endpoint.

Reply:
Resolved